src/plugins/autologin/MQ2AutoLogin.cpp

```diff
@@ -45,6 +45,96 @@ uint64_t ReenableTime = 0;
 int Level = -1;
 int Class = -1;
 
+class MQ2AutoLoginType* pAutoLoginType = nullptr;
+
+class MQ2AutoLoginType : public MQ2Type
+{
+public:
+	std::optional<ProfileRecord> m_record;
```

Comment:
datatype's should not hold state. this should be made a global variable instead, or it should be passed through the VarPtr

Reply:
Moved to global var.